test/utils/runners.go

```diff
@@ -1701,3 +1705,18 @@ func (config *DaemonConfig) Run() error {
 
 	return err
 }
+
+func NewWithControllerCreatePodStrategy(controllerName string, podTemplate *v1.Pod) TestPodCreateStrategy {
```

Comment:
We need to create replication controller for some test cases, so NewCustomCreatePodStrategy is not enough (it only creates the Pod).

Reply:
Do you know why we use replication controller? Why isn't creating pods enough?

Reply:
I am not sure:)

Reply:
I think using replication controller is just a shortcut to creating pods one by one. But since we already use NewCustomCreatePodStrategy, I think we should just use it.

Reply:
In the old benchmark test, `NewSimpleWithControllerCreatePodStrategy` was called to create both replication controller and pods, not sure if the replication controller can affect scheduler logic.

Reply:
It shouldn't - scheduler only cares about pods.

Reply:
One way to verify is to get rid of the controller, run the new tests, and compare with old tests.
In fact, we should do the comparison anyways, to make sure the new tests are consistent with old ones.

Please update the PR with the comparison so that we have confidence about the new tests.

Reply:
BTW, this is the command to run the old benchmarks if you haven's done so
```
make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''"  KUBE_TIMEOUT="--timeout=2h"  KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run='XXX' -bench=BenchmarkSchedulingPod.*Affinity"
```